otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/WebMvcMetricsCustomNameConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package com.opentable.metrics.micrometer;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.micrometer.core.instrument.config.MeterFilter;
+
+@Configuration
+public class WebMvcMetricsCustomNameConfiguration {
+
+    private static final String metricPrefix = "http.server.requests";
+    private static final String[] tagsToIgnore = {"exception", "outcome"};
```

Comment:
Why we want to ignore "exception"? We using it currently and it not causing cardinality problems,

Reply:
ok I will remove it

Reply:
Done